supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -1,6 +1,52 @@
 [[support]]
 = Support
 
+[[developer-preview-guidance]]
+== Developer Preview
+
+Developer Preview software provides early access to a technology, component, or feature, in advance of its possible inclusion in a Red Hat product offering. This enables customers to test the functionality and provide feedback during the development process. Documentation is not required for Developer Preview software. Documentation that is provided for Developer Preview software is subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
+
+[WARNING]
+====
+Some products, such as Openshift Container Platform,  do not include Developer Preview content in the documentation. Check with your Content Strategist or Support contact to confirm whether you can publish Developer Preview documentation for your product.
+====
+
+When documenting a Developer Preview software, follow these guidelines:
+
+* Add an admonition labeled **IMPORTANT** at the beginning of the Developer Preview content and include the template text.
+
+* Use initial uppercase capitalization, that is, Developer Preview.
+
+* Never use the phrase "supported as a Developer Preview", and avoid using "support" in Developer Preview descriptions. Instead, use neutral words like "available", "provide", "capability", and so on.
+
+* When the Developer Preview software becomes generally available, remove the IMPORTANT admonition from any document that includes content about the feature.
++
+[NOTE]
+====
+You might need to replace the Developer Preview admonition with a Technology Preview admonition. For more information, see <<Technology Preview>>.
+====
+
+Use the following template. Replace _<software_name>_ with the software name:
+
+.Example AsciiDoc: Developer Preview admonition template
+[source,text,subs="+quotes"]
+----
+[IMPORTANT]
+====
+_<software_name>_ is Developer Preview software only. Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview software for production or business-critical workloads. Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. This software might not have any documentation, is subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
```

Comment:
```suggestion
_<software_name>_ is Developer Preview software only. Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview software for production or business-critical workloads. Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering. Customers can use this software to test functionality and provide feedback during the development process. This software might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
```
- See my similar comments above about anthropomorphism and non-breaking spaces with Red Hat.
- Reword the penultimate sentence so that the items listed in the series are grammatically parallel.